lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+  def add_patron(name)
+    @patrons << name
+  end
+  def yell_at_patrons
+    @patrons.map! { |name| name.upcase }
```

Comment:
Nice job using `.map`. You wouldn't need to use the `!` operator here in this instance.